server/db/createTablesScript-base-example.sql

```diff
@@ -571,3 +571,38 @@ GRANT USAGE, SELECT ON SEQUENCE completed_activity_id_seq TO agora;
 
 CREATE INDEX IF NOT EXISTS idx_completed_activity_activity_id ON completed_activity (activity_id);
 
+
+-- Discussion API tables
+
+CREATE TYPE discussion_parents AS ENUM ('goal', 'topic');
+
+CREATE TABLE IF NOT EXISTS discussions (
+    parent_id INTEGER,
+    parent_type discussion_parents,
+    discussion_text VARCHAR,
+    CONSTRAINT discussion_parent PRIMARY KEY (parent_id, parent_type)
+);
+
+GRANT ALL PRIVILEGES ON TABLE discussions TO agora;
+
+CREATE TABLE IF NOT EXISTS discussion_comments (
+    comment_id SERIAL PRIMARY KEY,
+    parent_id INTEGER, 
+    parent_type discussion_parents,
+    comment_text VARCHAR,
+    creation_date TIMESTAMP DEFAULT current_timestamp,
+  	updated_date TIMESTAMP DEFAULT current_timestamp,
```

Comment:
Is this indentation intended?

Reply:
It is not intended, but I don't see it showing in my editor either... 
![image](https://user-images.githubusercontent.com/20839289/196197337-97dd0bd5-b1cc-4ae1-8a92-15c9b9eb9be8.png)